runtime/runtime/src/state_viewer/mod.rs

```diff
@@ -25,6 +25,9 @@ pub mod errors;
 pub struct TrieViewer {}
 
 impl TrieViewer {
+    /// Upper bound of the size of contract state that is still viewable.
+    const CONTRACT_STATE_SIZE_LIMIT: u64 = 50_000;
```

Comment:
How complicated is it to be exposed via the config.json file, so we can recommend people to run their own RPC node with the limit bumped to the values they feel is necessary?